tools/linter/clang_tidy/setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+if [ ! -d build ]; then
```

Comment:
While we're refactoring can we switch this to Python instead of mixing in Bash? Something like

```python
if not pathlib.Path("build").exists:
   subprocess.run([shutil.which("time"), sys.executable, ...])
   ...
```

Reply:
Sure, we can do that